service/src/main/java/com/codeforcommunity/database/table/ICommentTable.java

```diff
@@ -0,0 +1,30 @@
+package com.codeforcommunity.database.table;
```

Comment:
I think it makes sense to put the table classes in persist, that'll further help illustrate the MVC split

Reply:
They've been moved. I didn't have time to do it earlier since I merged in the persist PR like 10 minutes before the meeting.